batchHeader.go

```diff
@@ -295,7 +295,12 @@ func (bh *BatchHeader) CompanyDescriptiveDateField() string {
 
 // EffectiveEntryDateField get the EffectiveEntryDate in YYMMDD format
 func (bh *BatchHeader) EffectiveEntryDateField() string {
-	return bh.formatSimpleDate(bh.EffectiveEntryDate)
+	// ENR records require EffectiveEntryDate to be space filled. NACHA Page OR108
+	if bh.CompanyEntryDescription == "AUTOENROLL" {
+		return bh.alphaField("", 6) // YYMMDD
+	} else {
+		return bh.formatSimpleDate(bh.EffectiveEntryDate)
+	}
```

Comment:
This is a bit hackey, but I didn't think of a better way to ensure the files are written properly in this case.

Reply:
I don't see that as a hack.